<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WarlockTraining</class>
 <widget class="QDockWidget" name="WarlockTraining">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Warlock Training</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="nameLabel">
        <property name="text">
         <string>Platform Name:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="nameOutput">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Select a Platform</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="latitudeLabel">
        <property name="text">
         <string>Platform Latitude:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="longitudeLabel">
        <property name="text">
         <string>Platform Longitude:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="altitudeLabel">
        <property name="text">
         <string>Platform Altitude:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="headingLabel">
        <property name="text">
         <string>Platform Heading:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="wkf::LatitudeLineEdit" name="latitudeLineEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="wkf::LongitudeLineEdit" name="longitudeLineEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="wkf::AltitudeLineEdit" name="altitudeLineEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="wkf::AngleLineEdit" name="headingLineEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="1">
       <widget class="QPushButton" name="southPushButton">
        <property name="text">
         <string>South</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="westPushButton">
        <property name="text">
         <string>West</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="eastPushButton">
        <property name="text">
         <string>East</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="northPushButton">
        <property name="text">
         <string>North</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>wkf::AltitudeLineEdit</class>
   <extends>QLineEdit</extends>
   <header>WkfUnitTypes.hpp</header>
  </customwidget>
  <customwidget>
   <class>wkf::AngleLineEdit</class>
   <extends>QLineEdit</extends>
   <header>WkfUnitTypes.hpp</header>
  </customwidget>
  <customwidget>
   <class>wkf::LatitudeLineEdit</class>
   <extends>QLineEdit</extends>
   <header>WkfUnitTypes.hpp</header>
  </customwidget>
  <customwidget>
   <class>wkf::LongitudeLineEdit</class>
   <extends>QLineEdit</extends>
   <header>WkfUnitTypes.hpp</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>latitudeLineEdit</tabstop>
  <tabstop>longitudeLineEdit</tabstop>
  <tabstop>altitudeLineEdit</tabstop>
  <tabstop>headingLineEdit</tabstop>
  <tabstop>northPushButton</tabstop>
  <tabstop>westPushButton</tabstop>
  <tabstop>eastPushButton</tabstop>
  <tabstop>southPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
